netdata does not compile on FreeBSD 11.2-RELEASE-p4

--- plugins/freebsd.plugin/freebsd_sysctl.o ---
  CC       plugins/freebsd.plugin/freebsd_sysctl.o
In file included from plugins/freebsd.plugin/../../common.h:14:0,
                 from plugins/freebsd.plugin/plugin_freebsd.h:6,
                 from plugins/freebsd.plugin/freebsd_sysctl.c:3:
plugins/freebsd.plugin/freebsd_sysctl.c: In function 'do_net_inet_icmp_stats':
plugins/freebsd.plugin/../../database/rrd.h:702:85: error: expected expression before ',' token
     rrdset_create_custom(host, type, id, name, family, context, title, units, plugin, module, priority, update_every, chart_type, (host)->rrd_memory_mode, (host)->rrd_history_entries)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -340,7 +340,15 @@ set(WEB_PLUGIN_FILES
         web/server/web_client.h
         web/server/web_server.c
         web/server/web_server.h
-        web/server/single/single-threaded.c web/server/single/single-threaded.h web/server/multi/multi-threaded.c web/server/multi/multi-threaded.h web/server/static/static-threaded.c web/server/static/static-threaded.h web/server/web_client_cache.c web/server/web_client_cache.h)
+        web/server/single/single-threaded.c
+        web/server/single/single-threaded.h
+        web/server/multi/multi-threaded.c
+        web/server/multi/multi-threaded.h
+        web/server/static/static-threaded.c
+        web/server/static/static-threaded.h
+        web/server/web_client_cache.c
+        web/server/web_client_cache.h
+        )
 
 set(API_PLUGIN_FILES
         web/api/rrd2json.c

--- a/collectors/freebsd.plugin/freebsd_kstat_zfs.c
+++ b/collectors/freebsd.plugin/freebsd_kstat_zfs.c
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
 
 #include "plugin_freebsd.h"
-#include "../linux-proc.plugin/zfs_common.h"
+#include "collectors/proc.plugin/zfs_common.h"
 
 extern struct arcstats arcstats;
 

--- a/collectors/freebsd.plugin/freebsd_sysctl.c
+++ b/collectors/freebsd.plugin/freebsd_sysctl.c
@@ -2299,7 +2299,7 @@ int do_net_inet_icmp_stats(int update_every, usec_t dt) {
                             , NULL
                             , "IPv4 ICMP Errors"
                             , "packets/s"
-                              "freebsd.plugin",
+                            , "freebsd.plugin"
                             , "net.inet.icmp.stats"
                             , 2603
                             , update_every
@@ -2335,7 +2335,7 @@ int do_net_inet_icmp_stats(int update_every, usec_t dt) {
                             , NULL
                             , "IPv4 ICMP Messages"
                             , "packets/s"
-                              "freebsd.plugin",
+                            , "freebsd.plugin"
                             , "net.inet.icmp.stats"
                             , 2604
                             , update_every

--- a/collectors/freebsd.plugin/freebsd_sysctl.c
+++ b/collectors/freebsd.plugin/freebsd_sysctl.c
@@ -2266,7 +2266,7 @@ int do_net_inet_icmp_stats(int update_every, usec_t dt) {
                             , NULL
                             , "IPv4 ICMP Packets"
                             , "packets/s"
-                              "freebsd.plugin",
+                            , "freebsd.plugin"
                             , "net.inet.icmp.stats"
                             , 2602
                             , update_every

--- a/collectors/freebsd.plugin/plugin_freebsd.h
+++ b/collectors/freebsd.plugin/plugin_freebsd.h
@@ -3,7 +3,7 @@
 #ifndef NETDATA_PLUGIN_FREEBSD_H
 #define NETDATA_PLUGIN_FREEBSD_H 1
 
-#include "../../daemon/common.h"
+#include "daemon/common.h"
 
 #if (TARGET_OS == OS_FREEBSD)
 

--- a/web/server/web_client_cache.h
+++ b/web/server/web_client_cache.h
@@ -21,7 +21,6 @@ struct clients_cache {
 extern __thread struct clients_cache web_clients_cache;
 
 extern void web_client_release(struct web_client *w);
-extern void web_client_release(struct web_client *w);
 extern struct web_client *web_client_get_from_cache_or_allocate();
 extern void web_client_cache_destroy(void);
 extern void web_client_cache_verify(int force);

